### ISSUE TITLE
[UWP][Accessibility] Name is not defined for 'Close' control at left panel.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3832]
# Platform

* UWP

# Version of SDK

master

# Details

Repro Steps:
1. Launch the application.
2. New screen starts appearing. 
3. Navigate to various json file like Activity Update.JSON, Agenda.JSON etc. at left panel.
4. Navigate to 'Close' control appearing over json file like Agenda.JSON at left panel.
5. Verify name of 'Close' control by using Accessibility insights tools.

Actual Result: 
Name is not defined for 'Close' control appearing over json file like Agenda.JSON at left panel and due to this, screen reader user is unable to get the context of such control and won't be able to perform functionality over such control.

Expected Result: 
Name should be defined as 'Close' for 'Close' control appearing over json file like Agenda.JSON at left panel so that screen reader user could easily get the context of control and could be able to proceed further.
